docs/docs/StyleGuide.md

```diff
@@ -0,0 +1,581 @@
+---
+category: Documentation
+categoryindex: 1
+index: 6
+---
+# Style guide
+A comprehensive guide to F# Formatting Conventions
+===
+
+This article is written mostly based on ["F# Coding Guidelines"][1] (offline version) from Don Syme.
```

Comment:
This is the old style guide, it should not be listed in here.
What we want here is a page that explains why we are following https://docs.microsoft.com/en-us/dotnet/fsharp/style-guide/formatting and https://github.com/G-Research/fsharp-formatting-conventions.

How the proposal process over at https://github.com/fsharp/fslang-design#style-guide works.
How to contribute to the MS guide over at https://github.com/dotnet/docs.
(Example of that: https://github.com/dotnet/docs/pull/30094)